hack/generators/release-controllers/content/bump_versioned_resources.py

```diff
@@ -0,0 +1,79 @@
+
+import glob
+import os
+
+from shutil import copyfile
+
+
+def _get_next_version(version):
+    pieces = version.split('.')
+
+    if len(pieces) > 2:
+        print(f'Unable to determine next version number from version: {version}')
+        exit(1)
+
+    pieces[1] = str(int(pieces[1])+1)
+
+    return '.'.join(pieces)
+
+
+def _read_update_write_file(original_file, new_file, original_str, new_str):
+    with open(original_file, "r") as f:
+        data = f.read()
+
+    with open(new_file, "w+") as f:
+        f.write(data.replace(original_str, new_str))
+
+
+def _bump_image_file(current_version, new_version, image_files_path):
+    previous_file = image_files_path.joinpath(f'images-origin-{current_version}.yaml')
+    new_file = image_files_path.joinpath(f'images-origin-{new_version}.yaml')
+
+    _read_update_write_file(previous_file, new_file, current_version, new_version)
+
+
+def _bump_repo_files(current_version, new_version, repo_files_path):
+
+    for repo_file in glob.glob(f'{repo_files_path}/_repos/ocp-{current_version}-*.repo'):
+        bn = os.path.basename(repo_file)
+        new_repo_file = repo_files_path.joinpath(f'_repos/{bn.replace(current_version, new_version)}')
+        copyfile(repo_file, new_repo_file)
```

Comment:
Repo files have version specific strings in them. https://github.com/openshift/release/blob/734e2a8f1b0e928591c971f63a5812df5cdd30d2/core-services/release-controller/_repos/ocp-4.7-default.repo#L33